xc/device.ml

```diff
@@ -1912,6 +1912,30 @@ let stop ~xs ~qemu_domid domid  =
     stop_vgpu ();
     stop_qemu ()
 
+let is_hvm_linux domid =
```

Comment:
I'd be in favour of the result type, because the function may in future be used by other things, which may need to distinguish the "I don't know" case.